contentful_management/editor_interfaces_proxy.py

```diff
@@ -30,14 +30,21 @@ def _resource_class(self):
 
     def all(self):
         """
-        Gets the default Editor Interface.
+        Gets the default editor interface.
         """
 
         return super(EditorInterfacesProxy, self).all()
 
     def find(self):
         """
-        Gets the default Editor Interface.
+        Gets the default editor interface.
+        """
+
+        return self.all()
+
+    def default(self):
+        """
+        Gets the default editor interface.
```

Comment:
The 3 methods return a single `default` editor interface, it still has the 3 methods due to editor interfaces previously not being a singleton endpoint as it is now.